### DISCUSSION TITLE
In Search of a SandDance Freelancer.... Have an Idea and Need some Help.

### DISCUSSION THREAD

**Discussion by Procevent** [microsoft/SandDance#650] [Q&A]
Hello to all in the underworld!

I have joined this community by pulling the strings in search of a way to visualise, present and analyse data in my company, regarding the specific area that I manage which is company processes. We are an ISO certified organisation and have a lot of paper work, forms and registries that are currently unlinked to company processes. We are mapping out and managing the processes in order to streamline several activities throughout the company, linking different areas and documentation in a better way. This means that there are several stakeholders in a process and potentially several contributors to a document, or rather, different areas can provide input, development or receive output from a certain process in regard to events or documentation.

I have been pondering a way to visually represent these interactions for the layman user within the company, so he/she can understand which documents, procedures and/or processes his/her area is responsible for and to greater detail, which of these activities pertains specifically to their employee description: "What he/she has to do, how to do it and when to do it" in regard to company process. I drew out a 3D cuadrant on +X+Y to start out in which X axis are the type of document, procedure or process with a detail of name, company coding and Y cuadrant are the areas. This is the basic model im looking to set up, and from there expand to -X and -Y cuuadrant combinations.

I have researched and the best Im finding is a sand dance model, built via Power Bi, fed through an excel sheet database. I could do this through a 2D excel spreadsheet but am looking for something cooler, and more attractive for the user. It goes to say that this model should also provide for statystical process analysis of documents and areas interactions. 

But I have not found anyone who could be able to set this up for me... At the moment I have gone through Fiverr but have not found SandDance know how and that is why I am here, in search of guidance or advice as to who may know or is willing to freelance together to set up this model. I think this may be the right place to ask. My idea is for some one to help build the model through power Bi and set up a web link for user interaction, teach basic model management and then we then simply feed it the data. It would be a spot development with occasional consultancy.

Any ideas? Thanks before hand Gitters!

**Comment by VAEchompassion** [comment]
All you need is to have a virtual assistant. Check this one. https://arenaswendy330.wixsite.com/echompassion

**Comment by mmbconsultancy** [comment]
Hi there! I'm your dedicated Virtual Support Pro, ready to streamline your tasks and support your success. With my expertise in administrative support, organization, and multitasking, I'll help you reclaim valuable time and achieve your goals. From managing calendars and correspondence to research and data entry, I'll handle it all efficiently and with precision. Trust me to provide professional assistance, adapt to your needs, and enhance your productivity. Let's collaborate and make your workload lighter! Check my website https://caetemaryann.wixsite.com/mmbusinessconsultant
